mac/Keyman4MacIM/Keyman4MacIM/KMInputMethodAppDelegate.m

```diff
@@ -179,8 +180,18 @@ + (KMInputMethodAppDelegate *)AppDelegate {
 CGEventRef eventTapFunction(CGEventTapProxy proxy, CGEventType type, CGEventRef event, void *refcon) {
     KMInputMethodAppDelegate *appDelegate = [KMInputMethodAppDelegate AppDelegate];
     if (appDelegate != nil) {
+        if (type == kCGEventTapDisabledByTimeout || type == kCGEventTapDisabledByUserInput) {
+            // REVIEW: Not sure how *user* can disable the event tap, but in that case, should we *not* try to restart it?
```

Comment:
If Keyman won't work properly without the low level event tap, then we should try to re-enable in all circumstances. Can we update the comment accordingly? It's part of robustness. By "user" it probably doesn't mean a checkbox in UI but "anything other than timeout"?

Reply:
Also typo on next line.

Reply:
That's why it's in a REVIEW comment and not actually implemented in the code. If I can ever figure out how a user could (knowingly or unknowingly) disable it, I'd want to have another look, because if it's just going to come right back and disable it again, we need to figure out how to make the fallback code as robust as possible. I thought about putting in some kind of counter/flag to ensure that the very next event is not another disable.